internal/home/testdata/12to13-regression.yaml

```diff
@@ -0,0 +1,3 @@
+schema_version: 12
```

Comment:
Here, too, a TODO should be added to make it a more faithful representation of a `config.yaml` at version `12`.  With YAML, you can never be too defensive, heh.